lib/rubygems.rb

```diff
@@ -583,17 +583,8 @@ def self.find_latest_files(glob, check_load_path=true)
 
   def self.find_home
     windows = File::ALT_SEPARATOR
-    if not windows or RUBY_VERSION >= '1.9' then
-      File.expand_path "~"
-    else
-      ['HOME', 'USERPROFILE'].each do |key|
-        return File.expand_path ENV[key] if ENV[key]
-      end
 
-      if ENV['HOMEDRIVE'] && ENV['HOMEPATH'] then
-        File.expand_path "#{ENV['HOMEDRIVE']}#{ENV['HOMEPATH']}"
-      end
-    end
+    File.expand_path "~"
```

Comment:
Is there any reason we can't just use `Dir.home` here?

Reply:
`Dir.home` was provided by Ruby 1.9.2+. I will replace it.